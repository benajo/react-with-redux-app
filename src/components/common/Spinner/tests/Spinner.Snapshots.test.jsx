import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from '../Spinner';

it('render Spinner component', () => {
  const tree = renderer.create(<Spinner />);
  expect(tree).toMatchSnapshot();
});
