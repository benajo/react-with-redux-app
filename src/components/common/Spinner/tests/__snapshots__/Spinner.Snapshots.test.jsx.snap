// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Spinner component 1`] = `
<div
  className="loader"
>
  Loading...
</div>
`;
